pkg/virt-controller/watch/migration.go

```diff
@@ -273,13 +281,16 @@ func (md *MigrationController) execute(key string) error {
 			}
 			return setMigrationFailed(migration)
 		case k8sv1.PodSucceeded:
+			eventMsg := fmt.Sprintf("VM migrated from %s to %s", vm.Status.NodeName, targetPod.Spec.NodeName)
 			vm.Status.NodeName = targetPod.Spec.NodeName
 			vm.Status.MigrationNodeName = ""
 			vm.Status.Phase = kubev1.Running
 			if vm.ObjectMeta.Labels == nil {
 				vm.ObjectMeta.Labels = map[string]string{}
 			}
 			vm.ObjectMeta.Labels[kubev1.NodeNameLabel] = vm.Status.NodeName
+
+			md.recorder.Event(vm, k8sv1.EventTypeNormal, "Migrated", eventMsg)
```

Comment:
Can we make that constants? I would think of having three events. Something like that:

```golang
	FailedMigrateVirtualMachineReason = "FailedMigrate"
	SuccessfulMigrateVirtualMachineResaon = "SuccessfulMigrate"
       SuccessfullMigrateStartVirtualMachineReson = "SuccessfulMigrateStart"
```

Given that a migration can take ages, it makes sense to also have an event which indicates a migration start. What do you think?

Reply:
yup, i'll do this